model/config.go

```diff
@@ -1534,6 +1534,14 @@ func (s *MessageExportSettings) SetDefaults() {
 		s.ExportFromTimestamp = NewInt64(0)
 	}
 
+	if s.EnableExport != nil && *s.EnableExport && *s.ExportFromTimestamp == int64(0) {
+		// when the feature is enabled via the System Console, use the current timestamp as the start time for future exports
+		s.ExportFromTimestamp = NewInt64(GetMillis())
+	} else if s.EnableExport != nil && !*s.EnableExport {
+		// when the feature is disabled, reset the timestamp so that the timestamp will be set if the feature is re-enabled
+		s.ExportFromTimestamp = NewInt64(0)
+	}
```

Comment:
The control for this config value has been removed from the UI, so when the feature is enabled/disabled in the System Console or in the config file, this logic automatically sets the value of this timestamp. The timestamp controls what the creation time of the oldest post to export is, so this effectively tells the export job to start exporting posts that were made after the Message Export feature is first activated, and to ignore posts from before that time.